alt-fuel-finder/app/models/station.rb

```diff
@@ -1,6 +1,34 @@
 class Station
+  attr_reader :service
+  Station = Struct.new("Station",
+                      :id,
+                      :fuel_type_code,
+                      :station_phone,
+                      :city,
+                      :intersection_directions,
+                      :street_address,
+                      :fuel_type_code,
+                      :access_days_time,
+                      :distance
+                      )
```

Comment:
@josh-works Like to see students reaching for `Struct` for things. In this context it seems like using the Station class might be just as easier. It's common to use `Struct` when things are a work in progress. I tend to reach for it when I'm not sure what the class should look like.

I might do something like this:

```ruby
Person = Struct.new(:name)
person = Person.new("josh")
person.name # => "josh"
```